### DISCUSSION TITLE
Expand for View collection using UNION

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2690] [Q&A]
I'm trying to write a query that will combine two different tables. Do you think it's possible to use an expand and UNION query?

**Comment by ganigeorgiev** [comment]
You can check https://github.com/pocketbase/pocketbase/discussions/1938, but keep in mind that the fields will be resolved from an expression, meaning that id and `relation` fields for example will be detected as `json` and you'll not be able to use the `expand` parameter with it.
